feat: replace sass `@import` with `@use` and remove namespace

TheAlmightyCrumb opened the pull request.

matangeorgi reviewed (approved):
Amazing, really improves the experience in Codux.
Please make sure to cherry-pick this change into the `online` and `completed-lessons` branch.

TheAlmightyCrumb commented:
I changed the composition-task styles after merging from `main`. It used to have a root class with nested classes that are now separated, so I've changed them accordingly.
